python/BioSimSpace/Align/_align.py

```diff
@@ -168,17 +200,26 @@ def generateNetwork(molecules, names=None, work_dir=None, plot_network=False,
     _os.makedirs(work_dir + "/outputs", exist_ok=True)
 
     # Write all of the molecules to disk.
-    for x, molecule in enumerate(molecules):
-        _IO.saveMolecules(work_dir + f"/inputs/{x:03d}",
-            molecule, "pdb", property_map=property_map)
+    for x, (molecule, name) in enumerate(zip(molecules, names)):
```

Comment:
Yes, I think that Mol2 will be more robust in general. Glad the visualisation now works with Mol2 as well. I kept getting blank images when I tried it for some reason.

I do worry about conversions to Mol2 from other formats, e.g. if the original molecule was a PDB file. I think I'll update this after the merge so that we preferentially use Mol2, but write to PDB if that was the original format. (In this case, Mol2 bond-orders would be missing anyway.)

Reply:
agreed, I think in its current form nothing would break but the user would just not see bond orders when porting PDB files. IMO there should be a user-warning saying it's generally a bad idea to use PDB files for ligand structures.